plugins/reports/frontend/public/templates/vue-main.html

```diff
@@ -265,9 +265,11 @@ <h2> {{i18n('reports.title')}}
                 :key="idx"
                 v-for="(item, idx) in frequencyOptions"
             border>
-                {{item.label}}
-                <div class="text-small">
-                    {{item.description}}
+                <div class="bu-is-flex bu-is-justify-content-space-between">
+                    <span class="text-medium">{{item.label}}</span>
+                    <span class="text-small bu-mt-auto">
+                        {{item.description}}
+                    </span>
```

Comment:
this one is specifically for email reports but nothing generic? why? :slightly_smiling_face: